nntrainer/nntrainer_log.h

```diff
@@ -52,6 +52,33 @@
 #define ml_logd(...) \
   __android_log_print(ANDROID_LOG_DEBUG, TAG_NAME, __VA_ARGS__)
 
+#elif defined(_WIN32)
+#include <nntrainer_logger.h>
+
+#if !defined(ml_logi)
+#define ml_logi(...)                                                           \
```

Comment:
missing format ?

Reply:
If the format is included, an error occurs in cpp_checker and is being modified.